packages/matrix-resolve/src/index.test.ts

```
@@ -1,153 +1,140 @@
-          'm.server': '1.2.3.5:678'
-        }
-        ;(fetch as jest.Mock<any, any, any>).mockResolvedValue({
-          json: jest.fn().mockResolvedValue(wellKnown)
-        })
-        expect(await matrixResolve('matrix.org')).toBe('https://1.2.3.5:678/')
-      })
-
-      it('should accept IP address without port', async () => {
+          "m.server": "1.2.3.5:678",
+        };
+        (fetch as jest.Mock<any, any, any>).mockResolvedValue({
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)